packages/api/lib/GoogleOAuth2.js

```diff
@@ -76,5 +76,9 @@ class GoogleOAuth2 extends OAuth2 {
       username: userDataResponse.data.emails[0].value
     };
   }
+
+  async refreshAccessToken() {
```

Comment:
Per a conversation with @laurenfrederick, for now we're not going to implement token refresh for Google OAuth, since it's a minimally supported use case. Stubbing the function to just throw an error should result in a 500 response from the /refresh endpoint, which just means that after Google OAuth tokens in the dashboard expire, then refresh will fail and the user will be logged out.

I plan to test and verify this as well.

Reply:
I have manually tested and verified that using Google as the OAuth provider for the dashboard works with the new approach. I have also verified that token refresh for Google OAuth fails as expected and the user is logged out from the dashboard.

Note: If you use Google OAuth for authentication, you have to add the email for whatever user you're using to log in to the Dynamo users table.